src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionOptions.cs

```diff
@@ -13,19 +11,13 @@ namespace Microsoft.AspNetCore.ResponseCompression
     public class ResponseCompressionOptions
     {
         /// <summary>
-        /// Called when an HTTP request accepts a compatible compression algorithm, and returns True
-        /// if the response should be compressed.
+        /// Response Content-Type MIME types to compress.
         /// </summary>
-        public Func<HttpContext, bool> ShouldCompressResponse { get; set; }
+        public IEnumerable<string> MimeTypes { get; set; }
```

Comment:
Because ShouldCompressResponse moved to IResponseCompressionProvider and this was the only sane way I could find to flow mime types into there.